docs/common-issues.rst

```diff
@@ -0,0 +1,31 @@
+Common Issues with ev3dev-lang-python
+=====================================
+
+``/usr/bin/env: 'python3\r': No such file or directory``
```

Comment:
Another issue (possibly more common than this one) is "no module named ev3dev2". This is usually caused by users running the program on their computer instead of the EV3. 

e.g. https://github.com/ev3dev/ev3dev-lang-python/issues/539#issuecomment-435684710